packages/flutter/lib/src/foundation/collections.dart

```diff
@@ -4,16 +4,16 @@
 
 // TODO(ianh): These should be on the Set and List classes themselves.
 
-/// Compares two sets for deep equality.
+/// Compares two sets for shallow equality.
 ///
 /// Returns true if the sets are both null, or if they are both non-null, have
 /// the same length, and contain the same members. Returns false otherwise.
 /// Order is not compared.
 ///
-/// The term "deep" above refers to the first level of equality: if the elements
```

Comment:
I would replace "deep" here with "element-by-element equality"

Reply:
I would explain the term shallow with this sentence.